devDungeon/src/entities/BossAttackSkills.java

```diff
@@ -417,21 +415,20 @@ public static Skill normalAttack(int coolDown) {
                       () -> MissingComponentException.build(skillUser, PositionComponent.class))
                   .position();
           launchFireBall(bossPos, heroPos, bossPos, skillUser);
-          EventScheduler.getInstance()
-              .scheduleAction(
-                  () -> {
-                    Point heroPos2 = EntityUtils.getHeroPosition();
-                    if (heroPos2 == null) {
-                      return;
-                    }
-                    Vector2 heroDirection =
-                        new Vector2(heroPos2.x - heroPos.x, heroPos2.y - heroPos.y).nor();
-                    heroDirection.scl((float) (bossPos.distance(heroPos)) * 2);
-                    Point predictedHeroPos =
-                        new Point(heroPos2.x + heroDirection.x, heroPos2.y + heroDirection.y);
-                    launchFireBall(bossPos, predictedHeroPos, bossPos, skillUser);
-                  },
-                  50L);
+          EventScheduler.scheduleAction(
+              () -> {
+                Point heroPos2 = EntityUtils.getHeroPosition();
+                if (heroPos2 == null) {
+                  return;
+                }
+                Vector2 heroDirection =
+                    new Vector2(heroPos2.x - heroPos.x, heroPos2.y - heroPos.y).nor();
+                heroDirection.scl((float) (bossPos.distance(heroPos)) * 2);
+                Point predictedHeroPos =
+                    new Point(heroPos2.x + heroDirection.x, heroPos2.y + heroDirection.y);
```

Comment:
Urgs. Mal abgesehen davon, dass hier irgendwie die Abstraktion nicht stimmt ("es kratzt beim Lesen") - das ist Gefummel, was super in eine Hilfsmethode `predictHeroPosition(...)` versteckt werden könnte.

Abstraktion: Warum muss ich für einen neuen Vektor2 die Komponenten rausprökeln und darüber einen neuen Vektor2 erzeugen? Warum kann ich nicht einfach `Vektor2.of(heroPos2) - Vektor2.of(heroPos)` (jaja, wir sind in Java - also wenigstens `Vektor2.of(heroPos2).minus(Vektor2.of(heroPos))`) machen? 😆

Reply:
Es findest du alle meine Schönheitsfehler im DevDungeon :D Eigentlich will ich in den PR nur den EventScheduler verschieben. Ich stimmte dir sonst natürlich zu, das man hier den Code anpassen sollte. Aber hier sind jetzt nur Changes weil ich `.getInstance` vom EventScheduler entfernt habe.

Reply:
Mach ein Ticket auf und gut ist. Dieser PR geht ja nur um die Verschiebung ... :)

Reply:
#1927